spec/rspec/expectations/extensions/kernel_spec.rb

```diff
@@ -1,79 +0,0 @@
-RSpec.describe Object, "#should" do
-  before(:example) do
-    @target = "target"
-    @matcher = double("matcher")
-    allow(@matcher).to receive(:matches?).and_return(true)
-    allow(@matcher).to receive(:failure_message)
-  end
-
-  it "accepts and interacts with a matcher" do
-    expect(@matcher).to receive(:matches?).with(@target).and_return(true)
-    expect(@target).to @matcher
-  end
-
-  it "asks for a failure_message when matches? returns false" do
-    expect(@matcher).to receive(:matches?).with(@target).and_return(false)
-    expect(@matcher).to receive(:failure_message).and_return("the failure message")
-    expect {
-      expect(@target).to @matcher
-    }.to fail_with("the failure message")
-  end
```

Comment:
Covered in:
 - https://github.com/rspec/rspec-expectations/blob/bf9db76a293852a23ec4c46b55bf4ea58a66288c/spec/rspec/expectations/handler_spec.rb#L99
 - https://github.com/rspec/rspec-expectations/blob/bf9db76a293852a23ec4c46b55bf4ea58a66288c/spec/rspec/matchers/dsl_spec.rb#L825